src/database/ads.js

```diff
@@ -6,10 +6,8 @@ const collectionName = 'ads';
 
 async function insertAd(ad) {
   const database = await getDatabase();
-  const { insertedId } = await database
-    .collection(collectionName)
-    .insertOne(ad);
-  return insertedId;
+  const newElement = await database.collection(collectionName).insertOne(ad);
+  return newElement;
```

Comment:
now we can send the whole element on the response back to the client (so we can compare snapshots)